src/status_im/contexts/chat/events.cljs

```diff
@@ -431,3 +432,20 @@
                      [:chat/inputs current-chat-id :metadata :sending-image]
                      dissoc
                      (:uri original))})))
+
+(rf/reg-event-fx :chat.ui/show-profile
```

Comment:
Thanks for moving this!


btw - not related to your pr, but this events file is huge and we should really avoid adding files this large - they become very difficult to maintain. 
A good refactor here would be to categorise the events in this file and split them out further 👍

Reply:
I can create a cleanup issue for this file if that helps 👍

Reply:
That would be great, particularly if it can be as part of this work so it's not forgotten about:)

Need to clean the foundations a little

Reply:
yes @seanstrom please do. as this PR is a blocker for other works i want to merge it ASAP. so better move any refactoring to separate issue

Reply:
Oh yeah, was not suggesting to do it in this pr. just hinting for someone else to do it :)